mlflow/go/README.md

```diff
@@ -74,3 +74,11 @@ See the [pytest documentation](https://docs.pytest.org/en/8.2.x/how-to/usage.htm
 ## Supported endpoints
 
 The currently supported endpoints can be found in [mlflow/tools/generate/endspoints.go](./tools/generate/endspoints.go).
+
+## Linters
+
+We have enabled various linters from [golangci-lint](https://golangci-lint.run/), you can run these via:
+
+```bash
+golangci-lint run ./mlflow/go/...
```

Comment:
```suggestion
golangci-lint run
```

Reply:
I slightly disagree here, is letting it loose on the root folder not going to try and find `*.go` files in all folders? Perhaps not noticeable today but you should really only lint our own Go code.

Reply:
And also `pre-commit run golangci-lint --all-files` to run on all the files, and not just the staged ones.

Reply:
I'm just somewhat worried that some unexpected folders will at some point contain Go code.
For example, does it skip `node_modules` under `mlflow/server/js`?

Reply:
that's something we should then configure in `.pre-commit-config.yaml`, as the CI _will_ fail otherwise